front/src/views/EditStandardProduct/EditStandardProductView.tsx

```diff
@@ -0,0 +1,177 @@
+import { Suspense, useCallback } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+import { useMutation, useSuspenseQuery } from "@apollo/client";
+import { useAtomValue } from "jotai";
+import { ErrorBoundary } from "@sentry/react";
+import { Box, Center, Heading } from "@chakra-ui/react";
+
+import { MobileBreadcrumbButton } from "components/BreadcrumbNavigation";
+import { graphql } from "../../../../graphql/graphql";
+import { AlertWithoutAction } from "components/Alerts";
+import { FormSkeleton } from "components/Skeletons";
+import { selectedBaseIdAtom } from "stores/globalPreferenceStore";
+import { useNotification } from "hooks/useNotification";
+import { useErrorHandling } from "hooks/useErrorHandling";
+import {
+  findDefaultValues,
+  standardProductRawToFormDataTransformer,
+} from "views/EnableStandardProduct/components/transformer";
+import EditStandardProductForm, {
+  EditStandardProductFormOutput,
+} from "./components/EditStandardProductForm";
+import { STANDARD_PRODUCT_QUERY } from "queries/queries";
+
+export const StandardProductQueryErrorText =
+  "Could not fetch ASSORT standard data! Please try reloading the page.";
+
+export const EDIT_STANDARD_PRODUCT_MUTATION = graphql(
+  `
+    mutation EditStandardProductMutation(
+      $id: ID!
+      $comment: String
+      $inShop: Boolean
+      $price: Int
+    ) {
+      editStandardProductInstantiation(
+        editInput: { id: $id, comment: $comment, inShop: $inShop, price: $price }
+      ) {
+        __typename
+        ... on Product {
+          id
+          type
+          standardProduct {
+            id
+          }
+          comment
+          inShop
+          price
+          createdOn
+          createdBy {
+            id
+            name
+          }
```

Comment:
rather than the created fields query `lastModifiedOn`and `lastModifiedBy` because this will automatically updated the ManageProducts table

Reply:
Also in EditCustomProductView please add the lastModified* fields to the EDIT_CUSTOM_PRODUCT_MUTATION